### DISCUSSION TITLE
Bulk JSON files for court opinions in Zip file format

### DISCUSSION THREAD

**Discussion by deantaylormax** [freelawproject/courtlistener#3417] [General]
I used to be able to download, up to the previous month, all cases from Ohio courts.  The API documentation does not appear to allow that.  Am I missing something?

**Comment by mlissner** [comment]
Oh, it's been quite a while since we provided that, I'm afraid. As we grew, generating those JSON-based files became harder and harder, and we dropped that format, replacing it with the one we have now, which is bigger, but also harder to deal with. 

We have an open issue to do a dump just for SCOTUS, but I don't think we'll be bringing back our court-based dumps beyond that. There are just simply too many courts and too much data these days. Generating JSON was particularly difficult. 

Here's the issue where the changes were made: https://github.com/freelawproject/courtlistener/issues/1983

And the issue that's planning a SCOTUS file: https://github.com/freelawproject/courtlistener/issues/3072
